cmd/ak/cmd/sessions/start.go

```diff
@@ -15,70 +14,53 @@ import (
 )
 
 const (
-	defaultWait = 1 * time.Second
+	defaultPollInterval = 1 * time.Second
 )
 
 var (
-	entryPoint   string
-	memos        []string
-	wait         bool
-	waitInterval time.Duration
+	entryPoint string
+	memos      []string
+	build      string
+	deployID   string
 )
 
 var startCmd = common.StandardCommand(&cobra.Command{
-	Use:   "start <--deployment-id=...> <--event-id=...> <--entrypoint=...> [--memo=...] [--wait]",
+	Use:   "start [--build-id=...] [--env=...] [--deployment-id=...] <--entrypoint=...> [--memo=...] [--watch] [--watch-timeout=...] [--poll-interval=...] [--no-timestamps] [--quiet]",
```

Comment:
it feels that according to the command description all 3 are optional: bid, env, did
maybe < did | bid env> ?
is env optional if bid is specified?

Reply:
going to merge this, @daabr i'll fix this according to whatever you say